src/20230907_AaveV3_Multi_FreezeStewards/FreezeStewards.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Freeze Stewards"
+author: "BGD Labs"
+discussions: ""
+---
+
+## Simple Summary
+As the Aave V3 Ethereum market already has a freezing steward, which allows the emergency admin to freeze reserves, this AIP aims to synchronize the behavior by doing the same across networks. 
+
+## Motivation
+In order to maintain security across Aave V3 deployments, it's essential to maintain up-to-date preventative functionality on all networks.
+
+## Specification
+Adds a `FreezingSteward` as the `riskAdmin` on the canonical Aave V3 deployments on the following networks:
+
+- Optimism
+- Arbitrum
+- Avalanche
+- Polygon
+- Metis
+- Base
+
+The `FreezingSteward` is identical to the one currently deployed on Ethereum, and only allows the `emergencyAdmin` listed in the `ACLManager` for the given deployment to freeze reserves.
```

Comment:
You can add link to Etherscan on the FreezingSteward

Reply:
Added an Etherscan link for the Ethereum deployed FreezingSteward!